.github/workflows/php-test.yml

```diff
@@ -50,7 +58,7 @@ jobs:
       - name: Setup PHPCS
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '7.4'
+          php-version: '8.1'
```

Comment:
Considering it's marked experimental above for phpunit, this should probably stay at 7.4. Same for phpstan.